travelwise/api/views.py

```diff
@@ -119,7 +119,16 @@ def chat(request):
     # Prevent OpenAI API calls when testing
     # return JsonResponse({ 'message': 'Test Response' })
     
-    def create_activity(function_arguments):
+    response = {
+        'activities': {
+            'created': None,
+            'deleted': None,
+            'modified': None,
+        },
+    }
+    response_activity_created = None
```

Comment:
Are you having everything in memory since you storing the responses for each function in this dict?

Reply:
The activities field is how activities were changed, so not all activities are loaded. Messages are buffered through.